.github/workflows/pr.yml

```
@@ -19,8 +19,20 @@ jobs:
     # One runner for each domain
     strategy:
       matrix:
-        domain: [blas, dft, lapack, rng]
-    name: MKL ${{ matrix.domain }} CPU
+        include:
+        - config: portBLAS
+          options: -DENABLE_PORTBLAS_BACKEND=ON -DENABLE_MKLCPU_BACKEND=OFF -DTARGET_DOMAINS=blas -DCMAKE_CXX_FLAGS="-fsycl -fsycl-targets=spir64"
+        - config: portFFT
+          options: -DENABLE_PORTFFT_BACKEND=ON -DENABLE_MKLCPU_BACKEND=OFF -DTARGET_DOMAINS=dft -DCMAKE_CXX_FLAGS="-fsycl -fsycl-targets=spir64"
```

```suggestion
          options: -DENABLE_PORTFFT_BACKEND=ON -DENABLE_MKLCPU_BACKEND=OFF -DENABLE_MKLGPU_BACKEND=OFF -DTARGET_DOMAINS=dft
```
Same comment than above.

---

Resolving this comment as we need to fix the portFFT backend to not use `nvptx64-nvidia-cuda` if the CUDA runtime is not installed.